platforms/xamarin.mac/Facades/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -0,0 +1,4 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Are these interesting to keep?

Reply:
No, we can remove these facades now that the types are in corlib.